### ISSUE TITLE
Add custom number of observation bias estimation per arc

### ISSUE THREAD

**Issue by habdolhay** [Tudat/tudat#307]
As of now, if one requires to perform the estimation process in a single arc, the observation biases can only be estimated once in that one arc. So it is not possible to do multiple observation biases estimation in a single arc.

**Comment by DominicDirkx** [comment]
Hi Hamid,

I've written the code for this and pushed it to a branch here:

https://github.com/DominicDirkx/tudat/tree/ArcWiseBiasEstimation

I haven't had the chance to add much comments/documentation yet. Once this is done, I'll push it to the Tudat master. If you want to start using this before then, let me know and I'll post a short explanation here,

Dominic

**Comment by DominicDirkx** [comment]
This code has now been merged into the master branch.
